meson.build

```diff
@@ -860,6 +860,8 @@ have = false
 if localegen_path != ''
         conf.set_quoted('LOCALEGEN_PATH', localegen_path)
         have = true
+        locale_path_writeable = ',' + libdir + '/locale'
```

Comment:
Shouldn't this be a space separated list?

Reply:
Ah, right... Stupid me.